utils/image.go

```diff
@@ -51,14 +78,26 @@ func ExtractOCIImage(targetImage, targetDestination, targetPlatform string, isLo
 }
 
 // image returns the proper image to pull with transport and auth
-func getimage(ref name.Reference, platform v1.Platform, local bool) (v1.Image, error) {
-	if local {
-		return daemon.Image(ref)
+// tries local daemon first and then fallbacks into remote
+func getimage(ref name.Reference, platform v1.Platform) (v1.Image, error) {
+	var image v1.Image
+	var err error
+	tr := transport.NewRetry(http.DefaultTransport,
+		transport.WithRetryBackoff(defaultRetryBackoff),
+		transport.WithRetryPredicate(defaultRetryPredicate),
+	)
+
+	image, err = daemon.Image(ref)
+	fmt.Println("lo")
```

Comment:
```suggestion
```

Reply:
dropped them directly on master, sorry about this :sob: